lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_merge_strings_files.rb

```diff
@@ -40,8 +41,8 @@ def self.available_options
           FastlaneCore::ConfigItem.new(
             key: :paths,
             env_name: 'FL_IOS_MERGE_STRINGS_FILES_PATHS',
-            description: 'The paths of all the `.strings` files to merge together',
-            type: Array,
+            description: 'A hash of the paths of all the `.strings` files to merge together, with the prefix to be used for their keys as associated value',
+            type: Hash,
```

Comment:
Note that this is source-breaking and will require a major version bump (`4.0.0`).

I've considered trying to support both `Array` and `Hash`, which would mean that the old API would still work, and call site would be simpler to use in case you don't need any prefix (like WCiOS).

For the time being I've judged that, given that new action is not used in any of the client repos yet, better keep a simple and consistent API rather that try to support both `Array` and `Hash` for flexibility and for avoiding breaking change.

I could have my mind changed if needed — that would just require us to use `skip_type_validation: true` on that `ConfigItem` and do the check ourselves to accept both `Hash` and `Array`, and then do some `paths = paths.map { |f| [f, nil] }.to_h if paths.is_a?(Array)` in the action's body… not the end of the world, just not sure it's worth it?